### DISCUSSION TITLE
non English in Browse For Folder

### DISCUSSION THREAD

**Discussion by metallicblue3000** [enzo1982/freac#495] [Help & Support] [answered]
When selecting output directory there is some non English on English UK Windows 10

![freac output selection](https://github.com/enzo1982/freac/assets/135355096/a70de680-cb44-4e58-8aef-33ed7953da1d)

**Comment by enzo1982** [comment]
Hi, I cannot reproduce this here. Looks like it's accessing some invalid memory location and interpreting it as a Unicode string, but I'm not sure how that would happen in this situation. Does the displayed text change when you try again after restarting the program?

**Reply by metallicblue3000** [reply]
When I tried it again just after installing I kept getting the same result as the first post. But today I turned my PC on from shutdown and tried selecting the ouptput folder again:

![select first open](https://github.com/enzo1982/freac/assets/135355096/b4b0df1d-65bd-475e-a8f7-f4ba046a0442)

Then after canceling and clicking select again I get:

![select 2nd open](https://github.com/enzo1982/freac/assets/135355096/73ce590c-15e5-46c9-aaa2-4c5f47341b0a)

And I get that again if I cancel and select again. If I close the program and then reopen it I get the first image again and then the second if I cancel and select again.

I got fre:ac though the Microsoft store. Should I be worried I got a bad copy. I've also made some files using fre:ac should I be worried about them? It seems to work apart from this and I had been able to select the output folder even though I could not read the text at the time.

**Reply by enzo1982** [reply, marked as answer]
Thank you for providing these details! Indeed, I can reproduce the issue now using the version from the Microsoft Store. The behavior is the same that you describe. After starting the app, I sometimes get the Chinese characters the first time I open that dialog, but get the correct text after that.

The binaries of the Microsoft Store version are identical to the regular download version (I checked just now), so there's nothing wrong with those. However, Windows Apps (installed from the Microsoft Store and placed in the `C:\Program Files\WindowsApps` folder) can sometimes exhibit slightly different runtime behavior compared to the same binaries run as regular desktop apps.

So no, you don't have a bad copy of fre:ac and I wouldn't worry about files you created with it. Most likely this issue affects only that dialog. I will try and see if I can track down what exactly is causing this and provide a fix in a future update.

**Reply by metallicblue3000** [reply]
Thank you for looking into it and replying to me. I'll keep using fre:ac.
